### ISSUE TITLE
Can not set global disk options

### ISSUE THREAD

**Issue by kvaps** [LINBIT/linstor-server#4]
# Bug

According this mail:
http://lists.linbit.com/pipermail/drbd-user/2018-July/024228.html

Global disk options can be set, but they art not working.

Example: the `net` options can be set and they are working fine:
```
max-buffers, protocol, rcvbuf-size, sndbuf-size
```

But `disk` options can be set,  and the are not working afterwards:
```
c-fill-target, c-max-rate, c-plan-ahead, c-min-rate`
```

## Steps to reproduce
* Run 
  ```
  linstor c drbd-options --c-plan-ahead=10 --c-min-rate=$((20*1024)) --c-max-rate=$((720*1024)) --c-fill-target=$((10*1024))
  ```
*  Check your `/var/lib/drbd.d/linstor_common.conf`

**Comment by raltnoeder** [comment]
Scheduled for further analysis. Thanks for reporting!

**Comment by rp-** [comment]
This has been fixed internally and will be in the next release

**Comment by rp-** [comment]
should be fixed in v0.5.0

**Comment by kvaps** [comment]
agree working fine for now, there is only need to disable user security in controller
https://github.com/LINBIT/linbit-documentation/blob/master/UG9/en/administration-linstor.adoc#disable-user-security
